python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -184,7 +184,7 @@ class Job(GangaObject):
 
     _schema = Schema(Version(1, 6), {'inputsandbox': FileItem(defvalue=[], typelist=['str', 'Ganga.GPIDev.Lib.File.File.File'], sequence=1, doc="list of File objects shipped to the worker node "),
                                      'outputsandbox': SimpleItem(defvalue=[], typelist=['str'], sequence=1, copyable=_outputfieldCopyable(), doc="list of filenames or patterns shipped from the worker node"),
-                                     'info': ComponentItem('jobinfos', defvalue=None, doc='JobInfo '),
+                                     'info': ComponentItem('jobinfos', defvalue=JobInfo(), doc='JobInfo '),
```

Comment:
And again I'm getting the strong opinion that this is a mess to get the default value for an object over just writing some python code.
I don't know where or how the category will decide that a JobInfo is the best thing to put here as there is simply no mention of a category or equivalent.

This is done elsewhere for the JobTime etc so I'm sticking with this if the object isn't being correctly assigned. Do we ever expect this not to be valid?